<?xml version="1.0"?>
<launch>
    <!--<env name="ROSCONSOLE_CONFIG_FILE" value="/home/szj/catkin_ws/src/nwpu_explorer/explorer_vision/yolotest/rosconsole.config"/>-->
    <!-- ATTENTION: There many modes need image information!!! But you can only use one of them by a node-->
    <arg name="pub_qrcode_topic" default="/image_percept" />
   <!-- <arg name="sub_image_qrcode_topic_camera1" default="/camera_left/image_raw" /> -->
    <!--my add-->
    <!-- <arg name="sub_image_qrcode_topic_camera2" default="/camera_right/image_raw" /> -->
    <arg name="sub_image_qrcode_topic_camera2" default="/camera_left/image_raw/compressed" />
    <arg name="sub_image_qrcode_topic_camera1" default="/camera_right/image_raw/compressed" />
    <arg name="camera_info1" default="/camera_right/camera_info" />
    <arg name="camera_info2" default="/camera_left/camera_info" />

    <node pkg="yolotest" name="yoloTest_detection" type="yolotest_node">
        <!--The topic that could added in the map-->
        <param name="pub_qrcode_topic" value="$(arg pub_qrcode_topic)" />
        <!-- The topic that Qrcode was devided , and be published-->
        <!-- The topic that this program needed -->
        <param name="sub_image_qrcode_topic_camera1" value="$(arg sub_image_qrcode_topic_camera1)" />
        <param name="sub_image_qrcode_topic_camera2" value="$(arg sub_image_qrcode_topic_camera2)" />
        <param name="camera_info1" value="$(arg camera_info1)" />
        <param name="camera_info2" value="$(arg camera_info2)" />
    </node>


</launch>